app/assets/stylesheets/show.scss

```diff
@@ -0,0 +1,76 @@
+@import "foundation_and_overrides";
+@import "index";
+
+.art-details {
+  margin: auto;
+  margin-top: 20px;
+  margin-bottom: 35px;
+  display: block;
+  width: 50%;
+  background-color: #088da5;
+  box-shadow: 5px 5px 3px black;
+  li {
+    list-style-type: none;
+    font-size: 1.5em;
+    color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.